cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -29,16 +34,20 @@
                               search_types=settings.search_types,
                               search_size=settings.search_size)
 %}
+{% set hide_header_slug = settings.hide_header_slug or false %}
```

Comment:
Why does it need to be pulled out at all? Is it just to shorten the name when used on L:42?